telemetry/definitions/commonDefinitions.json

```diff
@@ -2174,6 +2175,74 @@
                 }
             ]
         },
+        {
+            "name": "appBuilder_installTool",
+            "description": "Click Install Docker button in Getting Started Walkthrough",
```

Comment:
Should this reference Docker directly? Or should it just reference "Click install tool button"?